### ISSUE TITLE
Volume Not Starting at Set Value

### ISSUE THREAD

**Issue by McCoyCOREUSA** [File-New-Project/EarTrumpet#1495]
### Summary

When I start something, like Zoom, the volume always starts at 100%. The ET volume shows 35% (where I set it last), but any time I restart software or reboot the computer, the volume setting in ET doesn't apply until I manually move the slider again. Very frustrating. 

### Steps to reproduce

n/a

### EarTrumpet version

2.3.0.0

### Windows version

10.0.19045.3758

### Additional information

n/a

**Comment by riverar** [comment]
Agree. Windows is supposed to remember per-app audio levels but this hasn't worked properly in years and the audio team at Microsoft has become unresponsive to my reports.

One workaround is to use the [developer builds of EarTrumpet](https://eartrumpet.app) and use EarTrumpet Actions to set the app audio levels when the app starts.

**Comment by McCoyCOREUSA** [comment]
It doesn't work even when switching Zoom rooms. I go into a breakout room, or go back to the main Zoom room, and the volume goes back to 100%. Kind of makes this app useless... and I paid for it.

**Comment by riverar** [comment]
EarTrumpet is free and open source, so not sure what you mean by "paid for it".

Maybe you can set up the audio levels in Zoom directly?

![Zoom-Volume-Sliders-in-Audio-Settings-2272472308](https://github.com/File-New-Project/EarTrumpet/assets/475132/a79a37fb-c6c8-430f-a361-30b6d15d3c71)

**Comment by McCoyCOREUSA** [comment]
My mistake. I thought I just purchased the upgrade. That was another software. Sorry.
